packages/domains-table/src/domains-table/style.scss

```diff
@@ -137,12 +137,39 @@
 		}
 	}
 
+	.domains-table-row__ssl-cell,
+	.domains-table-row__ssl-icon {
+
+		&__active {
+			color: var(--studio-green-50);
+			fill: var(--studio-green-50);
+		}
+
+		&__pending {
+			color: var(--studio-orange-50);
+			fill: var(--studio-orange-50);
+		}
+
+		&__disabled {
+			color: var(--studio-gray-50);
+			fill: var(--studio-gray-50);
+		}
```

Comment:
Do we know when a domain has `ssl_status: disabled`? I'm not sure, but if it indicates an error state, maybe red would be a more appropriate color.

Reply:
Excellent question. I've just done a bit of poking around (but pulled away before I fell all the way down the rabbit hole 😆) and didn't find an answer, but I _do_ know we don't allow site owners to disabled SSL themselves... so an error state feels most likely to me. Especially since the only site in my own account with SSL `disabled` is due to an expired domain.

@nuriapenya, any objection to red from a design perspective? I'd probably go with `studio-red-50`, because that's the color used for errors in the `status` column of this table. I worry a little that it might not have enough contrast with the `--studio-orange-50` we use for `pending`, but that also matches a similar state in the `status` column, so it's probably a broader conversation if it's an issue:

<img width="499" alt="image" src="https://github.com/user-attachments/assets/01344112-1ae8-4864-9294-cb78888da791">

In this screenshot, we can see SSL `Pending` in `orange-50`, `Disabled` in `red-50`. We can also see `red-50` in the Status column on the right.